1. Contributing/Google Summer of Code/README.md

```diff
@@ -1,5 +1,12 @@
 # Google Summer of Code 2017
 
+## How to apply
+
+Interested students should email the mentor(s) directly for discussion and further information.  You can also find them, most of the time, on our community server https://demo.rocket.chat/  for real-time chat.   Many of the mentors are fluent in both Portugeuese and English.  If you have ideas and proposals that is not on our idea list, or if a mentor is not available, please email to gsoc17@rocket.chat
```

Comment:
Yeah we'll need to hold off until we get that alias created or use `contact+gsoc17@rocket.chat`